src/work-allocation-2/tests/utils.spec.ts

```diff
@@ -31,6 +31,50 @@ export function getMockCaseworkers(): Caseworker[] {
   ];
 }
 
+/**
+ * Mock cases
+ */
+export function getMockCases(): Task[] {
```

Comment:
```suggestion
export function getMockCases(): Case[] {
```

And I think that'll then require changes to what actually gets returned...

Reply:
Changed.